Tests/API/ApiTradierTests.cs

```diff
@@ -0,0 +1,114 @@
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            _testAccount = Config.GetInt("job-user-id");
+            _testToken = Config.Get("api-access-token");
+            _dataFolder = Config.Get("data-folder");
+
+            _api = new Api.Api();
+            _api.Initialize(_testAccount, _testToken, _dataFolder);
+        }
+
+        /// <summary>
+        /// Live paper trading via Tradier
+        /// </summary>
+        [Test]
+        public void LiveEquityAlgorithms_CanBeUsedWithTradier_Successfully()
+        {
+            var refreshToken = Config.Get("tradier-refresh-token");
+            var account = Config.Get("tradier-account-id");
+            var accessToken = Config.Get("tradier-access-token");
+            var dateIssued = Config.Get("tradier-issued-at");
+
+            var file = new ProjectFile
+            {
+                Name = "main.cs",
+                Code = File.ReadAllText("../../../Algorithm.CSharp/BasicTemplateAlgorithm.cs")
+            };
+
+            // Create a new project
+            var project = _api.CreateProject($"Test project - {DateTime.Now.ToStringInvariant()}", Language.CSharp);
+
+            // Add Project Files
+            var addProjectFile = _api.AddProjectFile(project.Projects.First().ProjectId, file.Name, file.Code);
+            Assert.IsTrue(addProjectFile.Success);
+
+            var readProject = _api.ReadProject(project.Projects.First().ProjectId);
+            Assert.IsTrue(readProject.Success);
+
+            // Create compile
+            var compile = _api.CreateCompile(project.Projects.First().ProjectId);
+            Assert.IsTrue(compile.Success);
+
+            // Create default algorithm settings
+            var settings = new TradierLiveAlgorithmSettings(accessToken,
+                                                            dateIssued,
+                                                            refreshToken,
+                                                            account);
+
+            // Wait for project to compile
+            Thread.Sleep(10000);
+
+            // Create live default algorithm
+            var createLiveAlgorithm = _api.CreateLiveAlgorithm(project.Projects.First().ProjectId, compile.CompileId, "server512", settings);
+            Assert.IsTrue(createLiveAlgorithm.Success);
+
+            if (stopLiveAlgos)
+            {
+                // Liquidate live algorithm
+                var liquidateLive = _api.LiquidateLiveAlgorithm(project.Projects.First().ProjectId);
+                Assert.IsTrue(liquidateLive.Success);
+
+                // Stop live algorithm
+                var stopLive = _api.StopLiveAlgorithm(project.Projects.First().ProjectId);
+                Assert.IsTrue(stopLive.Success);
+
+                // Delete the project
+                var deleteProject = _api.DeleteProject(project.Projects.First().ProjectId);
+                Assert.IsTrue(deleteProject.Success);
+            }
```

Comment:
Pretty cool tests! 👍 They seem **very** similar though, I think we could create a base `ApiLiveTradingTests` (or similar) and abstract the core behavior into a method there?